docker/Dockerfile.backend

```
@@ -0,0 +1,13 @@
+FROM node:20-alpine
+
+WORKDIR /app
+
+COPY package.json package-lock.json ./
+
+RUN npm install --only=production
```

Líneas 5-7 El código del servidor debería copiarse manteniendo la estructura de directorios:
  ```suggestion
  COPY package.json package-lock.json ./

  RUN npm ci --only=production

  COPY server ./server
  ```